docs/src/content/docs/quickstart/sql-overview.mdx

```diff
@@ -0,0 +1,636 @@
+      <Query hidden={true} q="SELECT * FROM xt.txs ORDER BY tx_time DESC LIMIT 1" />
+    </Fiddle>
+    <Aside title="How to use XT Play">Feel free to edit any 'XT Play' SQL text input to experiment as you wish, but be aware:<br/><ul>
+<li>columns are ordered alphanumerically (this restriction will be lifted [soon](https://github.com/xtdb/xt-fiddle/issues/42))</li>
+<li>all changes are scoped to the particular instance in the page you are looking at and 'live' instances are completely unaware of each other (i.e. don't expect your changes to affect the 'state' of the database as you progress through the page).</li>
+</ul>You can click on the 'Open in xt-play' link beneath a given instance to see the full context and to experiment more freely. All XT Play instances execute statelessly, meaning a fresh database in the cloud is summoned into being every time you hit 'Run'!</Aside>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+To run your first INSERT transaction, simply enter the following SQL statement into the `psql` prompt, hit return, and you will see:
+
+```sql
+user=> INSERT INTO people (_id, name) VALUES (6, 'fred');
+INSERT 0 0
+```
+
+Note that XTDB doesn't currently return information about the number of rows inserted or modified by a statement when using `psql`.
+  </Card>
+  </TabItem>
+</Tabs>
+
+Here the `people` table has not been defined anywhere beforehand (i.e. there's no explicit `CREATE TABLE` [DDL](https://en.wikipedia.org/wiki/Data_definition_language) statement) - the table is created dynamically during the INSERT statement, along with any supplied columns and inferrable type information.
+
+Each record in XTDB must contain a user-provided `_id` primary key column, but other columns are fully dynamic based on the data that is inserted - meaning each row in XTDB can offer the flexibility of a document in a document database.
+
+The `_` prefix is convention for reserved columns and tables that XTDB handles automatically. For more information see [How XTDB works](/intro/data-model).
+
+### Query for that same row
+
+Querying this data back again is a simple matter of:
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Query q="SELECT * FROM people" />
+    </Fiddle>
+  </TabItem>
+  <TabItem label="psql" icon="laptop">
+  <Card>
+```sql
+user=> SELECT * FROM people;
+ _id | name
+-----+------
+   6 | fred
+(1 row)
+```
+  </Card>
+  </TabItem>
+</Tabs>
+
+### Evolve the table
+
+If we now INSERT another row with a slightly different shape, XTDB will gracefully evolve the schema of the `people` table to reflect the union of all the records it contains:
+
+<Tabs syncKey="psql">
+  <TabItem label="XT Play" icon="seti:db">
+    <Fiddle magicContext="my-context">
+      <Txs systemTime="2024-01-02"
+	   txs={`
+INSERT INTO people (_id, name, likes)
+  VALUES (9, 'bob', ['fishing', 3.14, {dynamic:'data'}])`}/>
```

Comment:
I would change this example to just add a new column or change a type.

We introduce complex data types in the next section.

Reply:
I did reflect on this but couldn't think of a concrete change to make that wouldn't make things somehow more complicated to look at (e.g. having an extra column on the table, or having to explain the json-ification of polymorphic columns in psql), worth circling back to sometime

Reply:
(suggestions welcome)